include/map.h

```diff
@@ -7,13 +7,16 @@
 #ifndef MAP_H_
 #define MAP_H_
 #ifdef _WIN32
-	#include <SDL.h>
+	#include "SDL2/SDL"
 	#include <sdl_ttf.h>
 	#include <SDL_image.h>
 #elif __APPLE__
 	#include <SDL2/SDL.h>
 #endif
 
+#define SDL_MAIN_HANDLED
```

Comment:
We already discuss that it is better you manage this part in Makefile. It is better because you don't need to change every thing.